bicycleparameters/io.py

```diff
@@ -109,6 +109,9 @@ def remove_uncertainties(dictionary):
             except TypeError:
                 # this is the case if the value is a float
                 noUncert[k] = v
+            except AttributeError:
+                # this is the case if the value is a float
+                noUncert[k] = v
```

Comment:
We should address this in a separate pull request. The precise issue needs to be identified and a unit test will need to be added.

Reply:
This is the one thing that hasn't been addressed. I'd like to know why this is needed.

Reply:
The app works fine with it removed. The issue must have been on my end back when I made the edit. It's all solved and removed now.